src/utils/emailReceipt.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html data-editor-version="2" class="sg-campaigns" xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+    <style type="text/css">
+      body, p, div {
+        font-family: inherit;
+        font-size: 14px;
+      }
+      body {
+        color: #000000;
+      }
+      body a {
+        color: #1188E6;
+        text-decoration: none;
+      }
+      p { margin: 0; padding: 0; }
+      table.wrapper {
+        width:100% !important;
```

Comment:
Good CSS practice generally avoids using `!important` since it overwrites all css from the top down instead of the bottom up. All elements that are within the table.wrapper element now have their properties affected depending on what the parent element's `!important` properties are. If, in the future, someone wanted to customize the inner width of elements, their changes wouldn't be taken into effect because `table.wrapper`'s width is `100%`. While this may not be a huge issue, this would definitely spark confusion for future devs who may wonder why their css isn't taken into effect. See if you can take out unnecessary instances of `!important` for future proof